stable/sysdig/templates/daemonset.yaml

```diff
@@ -1,8 +1,43 @@
 {{- if .Values.sysdig.AccessKey }}
+---
+{{- if .Values.kubernetes.rbac }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.sysdig.serviceAccountName }}
+  namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: sysdig-cluster-role
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+  - nonResourceURLs: ["*"]
+    verbs: ["*"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: sysdig-cluster-role-binding
+  namespace: {{ .Release.Namespace }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.sysdig.serviceAccountName }}
+    namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: sysdig-cluster-role
+  apiGroup: rbac.authorization.k8s.io
+{{- end }}
+---
 apiVersion: extensions/v1beta1
 kind: DaemonSet
 metadata:
   name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Explicit configuration of the namespace on the resources is not a pattern that's currently a part of the upstream charts repo and I think it could lead to confusion. I suggest we remove it unless there's a compelling reason. When I run an install with --namespace all resources are created in the correct namespace. Without it they end-up in default.